### PR TITLE
Split partial into separate words

### PR DESCRIPTION
**Pull request by i-like-robots**
The generated ngrams for the `searchLabel` field do not cross word boundaries so when the partial contains multiple words it will not match.

screen shot 2016-04-12 at 11 29 01

### CODE REVIEW

File: server/queries/search-query.js [hunk 1]
```diff
@@ -52,8 +52,10 @@ module.exports = (params) => {
 								must: (() => {
 									const bools = []
 
-									options.partial && bools.push({
-										term: { searchLabel: options.partial }
+									options.partial && options.partial.split(/\s/).map(item => {
+										bools.push({
```
Comment (andygnewman) [inline server/queries/search-query.js]: you could take account of accidental additional spaces, eg.
```
options.partial.trim().split(/\s+/)
```

Comment (i-like-robots) [inline server/queries/search-query.js]: In this case the partial will already be formatted: https://github.com/Financial-Times/next-tag-facets-api/blob/master/server/lib/format-partial.js#L9

Perhaps the partial formatter should return an array anyway?

Comment (andygnewman) [inline server/queries/search-query.js]: If the partial formatter is not used for any other use case, then returning an array sounds like a good idea.

Comment (i-like-robots) [inline server/queries/search-query.js]: Done!

**Pull request merged**